src/styles/global.css

```
@@ -238,61 +258,307 @@ a:focus-visible {
+  align-items: center;
+  min-height: 44px;
+  gap: 0.65rem;
+  color: var(--blue);
+  font-weight: 600;
+  white-space: nowrap;
+}
+.recommendations-feed svg {
+  width: 1.15rem;
+  height: 1.15rem;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)